tests/integration/conftest.py

```diff
@@ -31,7 +31,7 @@ def pytest_addoption(parser):
         '--work-dir', default='tests/tmp',
         help='Working directory for tests temporary files.')
     parser.addoption(
-        '--redis-up-timeout', default=3,
+        '--redis-up-timeout', default=120,
```

Comment:
why not to keep the default and change it only for the slow run?

Reply:
I think it is better to use a config that works for every test suite. Then, it will be easier to reproduce issues on our local. (No need to remember to pass extra config parameters)